db/migrate/20190722142115_change_foreign_key_for_vacancies.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeForeignKeyForVacancies < ActiveRecord::Migration[5.2]
```

Comment:
I'm not sure this is needed, as this column was only created for you locally.

Give me a shout and I can discuss this in more detail.